### DISCUSSION TITLE
SC Chat window stays stuck on loading when asking a question containing code

### DISCUSSION THREAD

**Discussion by avataraustin** [brianpetro/obsidian-smart-connections#523] [General]
I have noticed for a very long time since I started using SC, that if I ask a question of Chatgpt in the SC chat window and if my question contains computer code it will frequently (although not always) stall out and stay stuck on loading "..." I have to create a new chat with a different question in order to get a response.

dev log right after issue occurs:
![Screenshot 2024-03-27 at 12 35 02 PM](https://github.com/brianpetro/obsidian-smart-connections/assets/39737076/018984c4-9242-415d-8241-266faf7b3156)

**Comment by brianpetro** [comment]
@avataraustin, thanks for bringing this to my attention!

I started an issue to better track this here https://github.com/brianpetro/obsidian-smart-connections/issues/524 so I'm going to close this discussion.

If you could, please share an example of a chat input that fails in the new issue. That will help me be able to address this sooner.

Thanks
🌴
